third_party/tock/crates/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+# Copyright lowRISC contributors.
```

Comment:
Is there a particular reason for this being separate from the main rust directory?

Reply:
To keep tock deps namespaced separately from "host" rust deps.  Tock and libtock-rs aren't published in crates.io and I wanted to avoid the possibility of a name collision and make it obvious from the labels that certain deps were part of tock (ie: //third_party/tock/crates/...).